GOVERNANCE.md

```diff
@@ -0,0 +1,189 @@
+# Main Governance Document
+
+
+The Project
+===========
+
+The ArviZ Project (The Project) is an open source software project
+affiliated with the 501c3 NumFocus Foundation. The goal of The Project is to
+develop open source software and deploy open and public websites and services
+for reproducible, exploratory and interactive computing. The Software developed
+by The Project is released under the Apache 2 open source license,
+developed openly and hosted in public GitHub repositories under the
+[GitHub organization](https://github.com/arviz-devs/arviz). Examples of
+Project Software include the ArviZ code and the Documentation, etc. The Services run by the
+Project consist of public websites and web-services that are hosted
+at [https://arviz-devs.github.io/arviz/](https://arviz-devs.github.io/arviz/) 
+The Project is developed by a team of distributed developers, called
+Contributors. Contributors are individuals who have contributed code,
+documentation, designs or other work to one or more Project repositories.
+Anyone can be a Contributor. Contributors can be affiliated with any legal
+entity or none. Contributors participate in the project by submitting,
+reviewing and discussing GitHub Pull Requests and Issues and participating in
+open and public Project discussions on GitHub, Slack, and Gitter chat rooms. The foundation of Project participation is openness
+and transparency.
+
+There have many Contributors to the Project, whose contributions are listed in the logs of other repositories of the PyMC and projects.
+
+The Project Community consists of all Contributors and Users of the Project.
+Contributors work on behalf of and are responsible to the larger Project
+Community and we strive to keep the barrier between Contributors and Users as
+low as possible.
+
+The Project is formally affiliated with the 501c3 NumFOCUS Foundation
+([http://numfocus.org](http://numfocus.org)),  NumFOCUS is the
+only legal entity that has a formal relationship with the project (see
+Institutional Partners section below).
+
+### Governance
+
+This section describes the governance and leadership model of The Project.
+
+The foundations of Project governance are:
+
+-   Openness & Transparency
+-   Active Contribution
+-   Institutional Neutrality
+
+Traditionally, project leadership was unstructured but primarily driven
+by a subset of Core Contributors whose active and consistent
+contributions have been recognized by their receiving “commit rights” to the
+Project GitHub repositories. In general all Project decisions are made through
+consensus among the Core Developers with input from the Community. 
```

Comment:
I think all references use "core contributors" except this one. I prefer "core developers" or development team, which I also find to clarify the distinctions with general contributors by using a different name. However, core contributors is also ok. We should stick with the same name always though

Reply:
I agree we should stick to the same name. I think I remember that Core Contributors was used because it's defined in a larger sense than just the number of commits (which is closer to Core Dev definition).
It was in the gdoc but I didn't see it here. So I think this role should be clarified, as well as General Contributors, as Oriol wrote.

Reply:
Agree with consistent verbiage.

Two questions. I'll do one at a time.
The doc mostly uses contributors because "contributing" is a wider scope than "developing". We're inconsistent in the Google Doc as well, but to nail it down here. Developer or contributor? I'm in favor of contributor

Reply:
In favor of "Contributor" too, as it encompasses more what "contributing" means

Reply:
core contributor is prefectly fine too, I have edited my answers to reduce confusion